server/middleware/errorHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response, NextFunction } from 'express';
+import { CustomError, InternalError } from '../errors';
+
+interface OldError {
+  status: number;
+  log: string;
+  message: { err: string };
+}
+
+const errorHandler = (err: Error | OldError, _req: Request, res: Response, _next: NextFunction) => {
+  // If it is one of our custom errors use its properties/method
+  if (err instanceof CustomError) {
+    console.log(err.message);
```

Comment:
This looks great to me! I'm curious if 'console.error' would be more fitting here, however? Maybe not, just wondering.

Reply:
Great catch! `console.error` is def more appropriate.

I think I also need to be logging the entire err to get the stack trace for debugging. Refactor coming in hot